packages/cli/src/databases/entities/WorkflowHistory.ts

```diff
@@ -0,0 +1,28 @@
+import { Column, Entity, ManyToOne, PrimaryColumn } from 'typeorm';
+import { jsonColumnType } from './AbstractEntity';
+import { IConnections } from 'n8n-workflow';
+import type { INode } from 'n8n-workflow';
+import { WorkflowEntity } from './WorkflowEntity';
+
+@Entity()
+export class WorkflowHistory {
+	@PrimaryColumn({ length: 36 })
```

Comment:
Curious how did we settle on `36`?

Reply:
It's the length of UUIDs

Reply:
Sorry, followups: Why not use the length of nano IDs for `workflowId`? Why not use nanoIDs for `versionId`?

Reply:
Do we need to define the `length` here? what advantages does that have?

Reply:
@ivov The problem I think is because the migration to workflow ID from number to string was created with the UUID in mind, but then we settled on nanoid. It's not a problem since this is just an upper limit to the length, but does not take more space in the database.

The difference between `char` and `varchar` is that the latter uses `length + 1` bytes to delimit the end of the string, while the former means the database occupies always, regardless of the content length, the same space, basically "filling it up with empty stuff" so the table is uniform.

@netroy I don't think there's any advantage from typeorm's perspective, but it's a good hint for anyone using the code that this is the expected db length, so it's more of a QoL.

Reply:
looking at typeorm code, it seems like `length` from decorators is only used if you want to let typeorm create the tables, instead of using migrations. perhaps we should look into removing metadata like this, it doesn't really serve any purpose.
If we want to impose length validations, then we need to use `@Length` from `class-validator` instead.

Reply:
Makes sense, I think it would be out of the scope of this PR to remove this `length` metadata from the rest of the files; Still I think it doesn't hurt and helps to have a place to check for field length. The problem is if this becomes inconsistent with db schema, so I see a point for removing it.